Ginger/Ginger/Actions/ActionEditPages/ActDataSourcePage.xaml.cs

```diff
@@ -375,6 +375,10 @@ public void SetDataSourceVEParams(string p)
                                             if (arrORCond[iOrCount].IndexOf("'") != -1)
                                             {
                                                 wColVal = arrORCond[iOrCount].Substring(arrORCond[iOrCount].IndexOf("'") + 1, arrORCond[iOrCount].LastIndexOf("'") - arrORCond[iOrCount].IndexOf("'") - 1);
+                                                if (wColVal.Contains("$$$"))
```

Comment:
what is the use of this "$$$" what we trying to achieve and what was the issue this code solves?

Reply:
In Data source Action whenever in where Clause if we are adding any variable after refreshing the value get Remove or only show the “{Var , Basically issue is we are splitting the where clause string with the space and in Variable it self have the space so Variable name also getting split by “ “.I have replaced the Variable space with the “$$$” then to show on UI I am just Replaced back  the “$$$” with the space to the Actual variable